File: 1446-angle-between-hands-of-a-clock/1446-angle-between-hands-of-a-clock.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        minuteAngle=(minutes/5)*30
        percentageMoved=(minutes/5)/12*(100)
        hoursHand=((hour%12)*30)
        hoursAfterExtra=hoursHand+30*percentageMoved/100
        angle=abs(hoursAfterExtra-minuteAngle)
        print(minuteAngle,percentageMoved,hoursHand,hoursAfterExtra,angle)
        return min(angle,360-angle)

File: 1446-angle-between-hands-of-a-clock/1446-angle-between-hands-of-a-clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        double minAngle=minutes*6;
        double hrsAngle=(hour%12)*30+(minutes/60.0)*30;
        double angle=Math.abs(hrsAngle-minAngle);
        return Math.min(360-angle,angle);
    }
}